### PR TITLE
fix: add scaleRuntime audit key

### PR DESCRIPTION
**Pull request by littlejiancc**
## What this PR does / why we need it:
fix: add scaleRuntime audit key

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)

## Which issue(s) this PR fixes:
[【审计日志】runtime扩缩容操作日志显示错误](https://erda.cloud/erda/dop/projects/387/issues/all?id=224203&issueFilter__urlQuery=eyJpdGVyYXRpb25JRHMiOls1MDZdLCJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIl0sImFzc2lnbmVlSURzIjpbIjEwMDEyNjEiXX0%3D&issueGantt__urlQuery=eyJ0b3RhbCI6MzQsInBhZ2VObyI6MSwicGFnZVNpemUiOjIwMCwiaXNzdWVWaWV3R3JvdXBWYWx1ZSI6ImdhbnR0IiwiSXNzdWVUeXBlIjoiQUxMIn0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTAwfQ%3D%3D&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=506&type=BUG)

Fixes #

### CODE REVIEW

File: shell/app/modules/org/common/audit-templates.json [hunk 1]
```diff
@@ -1790,6 +1790,17 @@
       "en": ""
     }
   },
+  "scaleRuntime": {
+    "desc": "服务扩缩容",
+    "success": {
+      "zh": "应用 [@appName](app) 中, 对 [@serviceName] 进行了扩缩容, [@scaleMessageZH]",
+      "en": "in app [@appName](app), Changed the scale of service [@serviceName], [@scaleMessageEN]"
```
Comment (sfwn) [inline shell/app/modules/org/common/audit-templates.json:1797]: Why `Changed` not `changed`

**Review by sfwn — CHANGES_REQUESTED** [review]

File: shell/app/modules/org/common/audit-templates.json (hunk 1 above)
Comment (littlejiancc) [reply shell/app/modules/org/common/audit-templates.json]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**